File: Data Structures & Algorithms/top-k-elements-in-list/submission-4.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = {}
        for num in nums:
            count[num] = count.get(num,0) + 1
        pq = []
        for key,v in count.items():
            heapq.heappush(pq, (v,key))
            if len(pq) > k:
                heapq.heappop(pq)
            
        ans = []
        for i in range(k):
            ans.append(heapq.heappop(pq)[1])
        return ans

File: Data Structures & Algorithms/find-duplicate-integer/submission-4.java
class Solution {
    public int findDuplicate(int[] nums) {
        for(int i=0; i<nums.length; i++) {
            int x = Math.abs(nums[i]);
            if(nums[x] < 0) return x;
            nums[x] = -nums[x];
        }
        return -1;
    }
}
